README.md

```diff
@@ -178,6 +184,8 @@ The variable type. Possible values:
 - `int`
 - `float`
 - `bool`
+- `timestamp`
+- `list`
```

Comment:
`list` is not a type. `list:{type}` is a modifier on other types. You would probably want to change this to `list:{type}` instead of `list`.